src/amber/dsl/server.cr

```diff
@@ -12,4 +12,10 @@ module Amber::DSL::Server
       end
     {% end %}
   end
+
+  macro namespace(scoped_namespace, value = :web)
```

Comment:
I swapped them because I want used it like this.
`
namespace "/foo" do
  namespace "/bar" do
  end
end

namespace "/foo1", :web do
  namespace "/bar" do
  end
end
`

Reply:
And what happen for 

```
routes :web do
  namespace "/foo", :api do
    get "/bar" ...
  end
end
```

This is inconsistant and allow developpers to do crappy things.